sale_delivery_rate/demo/delivery_carrier.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2017 LasLabs Inc.
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
```

Comment:
Ok. I'll switch these up. I think I've been using an older LasLabs module template that differs in a few small ways like this.